RibonDAO/core-api
# Tech 477: Pool balance control

# Description

- we need to know if it is possible to make a free donation based on the pool balance
- we can't just consult the wallet balance, because we use the batch system
- The idea is to consult this Pool Balance table, to check if there is fund, and on each donation we decrease the value of the balance
- The worker is needed, so we can update the balance with the wallet balance, just to check everyday

## Type of change

Please delete options that are not relevant.


## How to test

Put here all the things need to test this PR and verify your changes, also list any relevant details for tests (eg: have a wallet)

- Run the job and see the pool balance being created and updated
